analyzers/src/SonarAnalyzer.Vsix/SonarAnalyzer.CSharp.nuspec

```diff
@@ -6,7 +6,7 @@
     <title>SonarAnalyzer for C#</title>
     <authors>SonarSource</authors>
     <owners>SonarSource</owners>
-    <license type="file">License.txt</license>
+    <license type="expression">LGPL-3.0-only</license>
```

Comment:
I manually checked nuget.org UI and this translates into link https://licenses.nuget.org/LGPL-3.0-only

Reply:
nice!